### PR TITLE
(FE): code review change to Read more on EOSIO account creation

### LINKED ISSUE #359: Fix Broken links
links to eosio documentation are broken 

https://guide.eoscostarica.io/docs/eos-learn/accounts-and-permissions/

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #359

### Steps to test
1. Run `yarn start`
2. Go to `localhost:3000`
3. Go to Developer Guidelines > Learn EOSIO > Accounts And Permissions

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/eos-learn/accounts-and-permissions.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ An **account** on EOSIO is a legible name for humans that get stored on the bloc
 
 The account name must comply with certain requirements: characters from `a` to `z` in lower case, numbers from `1` to `5` and have 12 characters length. E. g. `eosioaccount`.
 
-Read more on [EOSIO accounts](https://eos-account-creator.com/choose/).
+Read more on [EOSIO account creation](https://eos-account-creator.com/choose/).
```
Comment (xavier506) [inline docs/eos-learn/accounts-and-permissions.md:13]: I thought we were going to update the link to the EOSIO documentation

Comment (xavier506) [inline docs/eos-learn/accounts-and-permissions.md:13]: I thought we were going to update the link to the EOSIO documentation https://developers.eos.io/manuals/eosjs/v21.0/how-to-guides/how-to-create-an-account

**Review by xavier506 — APPROVED** [review]
LGTM

**Comment by leisterfrancisco** [comment]
I didn't notice this check review before, my bad.

The used link is not the last one, I'm going to reopen the issue to make the update to the latest version.

**Pull request merged**